dockerfiles/platform-core/Dockerfile

```diff
@@ -0,0 +1,41 @@
+ARG BUILD_VER
+FROM hpccsystems/platform-build:${BUILD_VER} as build
+FROM ubuntu:18.04
+ENV DEBIAN_FRONTEND=noninteractive
+RUN apt clean && \
+    apt autoclean && \
+    apt install -f && \
+    apt autoremove && \
+    apt-get update
+
+RUN apt-get install -y \
+  xterm \
```

Comment:
why xterm?

Reply:
It's in our documentation as a requirement...

(Unlike most of the dependencies in the next section, which are NOT listed as requirements, but should be!)

Reply:
I haven't ever seen it as a requirement, I don't see it on the wiki.

Reply:
Hmmmm - maybe it was just in the docker file I used as my starting point... and the reasons for it being in there may be lost in the mists of time

I can try taking it out and see what happens.